cli/api/daemon.go

```diff
@@ -111,6 +112,7 @@ type daemon struct {
 	tokenExpiration  time.Duration
 
 	facade *facade.Facade
+	ssm    *servicestatemanager.BatchServiceStateManager
```

Comment:
Easy enough to do, but there are two arguments against that:
1. The facade member is the impl, not facade.FacadeInterface
2. ssm.SetFacade, ssm.Start, and ssm.Stop are not currently part of the interface, since a different implementation may not need those (or may have a different set of Facade methods it needs).  Of course, that's not actually going to happen ever.